test/api/user/GET_user_tags_id.test.js

```diff
@@ -0,0 +1,41 @@
+import {
+  generateUser,
+  requester,
+} from '../../helpers/api.helper';
+
+describe('GET /user/tags/id', () => {
+  let api, user;
+
+  beforeEach((done) => {
+    generateUser().then((usr) => {
+      user = usr;
+      api = requester(usr);
+      done();
+    }).catch(done);
+  });
+
+  it('gets a user\'s tag by id', (done) => {
+    api.get('/user/tags/' + user.tags[0].id)
+      .then((fetchedTag) => {
+        expect(fetchedTag).to.eql(user.tags[0]);
+        done();
+      })
+      .catch((err) => {
+        done(err);
+      });
+  });
+
+  it('fails for non-existent tags', (done) => {
+    api.get('/user/tags/not-an-id')
+      .then((fetched) => {
+        done(new Error('Request should have failed'));
+      })
+      .catch((err) => {
+        expect(err.code).to.eql(404);
+        done();
+      })
+      .catch((err) => {
```

Comment:
I think we need both of these, actually. The first one checks that the error code is correct (because the `api.get(...)` call should return a promise which errors), then the second one catches the error from the `expect(...)` line (if there is one) and passes it to `done`.